modules/rosa-cluster-autoscaler-cli-settings.adoc

```diff
@@ -0,0 +1,115 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa-cluster-autoscaling.adoc
+// * osd_cluster_admin/osd-cluster-autoscaling.adoc
+
+:_content-type: REFERENCE
+[id="rosa-cluster-cli-autoscale-settings_{context}"]
+= Cluster autoscaling settings using the ROSA CLI
+
+There are several options you can add to the cluster creation command to adjust autoscaling options. The tables explain all the configurable autoscaler settings when using the ROSA CLI (`rosa`).
+
+[cols="4",options="header"]
+|===
+|Setting
+|Description
+|Type or Range
+|Default
```

Comment:
Maybe instead of default we can have examples, it can be more clear and less committing